examples-crate/Cargo.toml

```diff
@@ -18,3 +18,7 @@ path = "render/main.rs"
 [[example]]
 name = "uv_mapping"
 path = "uv_mapping/main.rs"
+
+[[example]]
+name = "reinterpret"
+path = "reinterpret/main.rs"
```

Comment:
Could use a more descriptive name.

Reply:
What do you propose?

Reply:
Hmmmm... maybe `multiple_views`. Or mirror the name of the function and call it `with_cast`.